service/device/registry/test-steps/src/main/java/org/eclipse/kapua/service/device/registry/steps/BrokerSteps.java

```diff
@@ -515,6 +527,34 @@ public void disconnectClient(String clientName) throws Exception {
         }
     }
 
+    @When("I Force Disconnect connection with client id {string}")
+    public void disconnectConnectionWithClientId(String clientId) throws Exception {
+        DeviceConnection deviceConnection = deviceConnectionService.findByClientId(SYS_SCOPE_ID, clientId);
+        Assert.assertNotNull(deviceConnection);
+        logger.info("Found device connection for clientId {} - scope: {}, id: {}", clientId, deviceConnection.getScopeId(), deviceConnection.getId());
+
+        KapuaSecurityUtils.doPrivileged(() -> {
+            deviceConnectionService.disconnect(deviceConnection.getScopeId(), deviceConnection.getId());
+        });
+
+    }
+
+    @Then("Device(s) status is {string} within {int} second(s) for client id {string}")
+    public void deviceStatusIs(String expectedStatus, int timeout, String clientId) throws Exception {
+        String currentStatus = null;
+        while(!expectedStatus.equals(currentStatus) && timeout-->0) {
+            Thread.sleep(1000);
+
+            logger.info("Device(s) status countdown check: {}", timeout);
+            DeviceConnection deviceConnection = deviceConnectionService.findByClientId(SYS_SCOPE_ID, clientId);
```

Comment:
could I suggest to use a parameter to provide the scopeId? (so the test can be invoked also for children accounts)

Reply:
See above.